src/functions/weekly-check/index.ts

```diff
@@ -0,0 +1,6 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [  ],
```

Comment:
Same again here as for the update paid function, if there is no event how is this function called?